api/src/pcapi/core/offers/repository.py

```diff
@@ -466,14 +466,13 @@ def check_stock_consistency() -> list[int]:
     ]
 
 
-def find_tomorrow_event_stock_ids() -> set[int]:
-    """Find stocks linked to offers that happen tomorrow (and that are not cancelled)"""
-    tomorrow = datetime.now() + timedelta(days=1)
-    tomorrow_min = datetime.combine(tomorrow, time.min)
-    tomorrow_max = datetime.combine(tomorrow, time.max)
+def find_today_event_stock_ids() -> set[int]:
+    """Find stocks linked to offers that happen today (and that are not cancelled)"""
+    today_min = datetime.utcnow()
+    today_max = datetime.combine(today_min, time.max)
 
     stocks = (
-        Stock.query.filter(Stock.beginningDatetime.between(tomorrow_min, tomorrow_max))
+        Stock.query.filter(Stock.beginningDatetime.between(today_min, today_max))
         .join(Booking)
         .filter(Booking.status != BookingStatus.CANCELLED)
         .options(load_only(Stock.id))
```

Comment:
(détail) C'était déjà là avant ta PR, mais ce serait (un peu, probablement pas beaucoup) plus efficace de ne récupérer que les identifiants des Stock et de faire un distinct en SQL au passage.

```suggestion
        .with_entities(Stock.id).distinct()
```